shared/ethereum/deploy.go

```diff
@@ -42,27 +43,27 @@ type DeployedContracts struct {
 // DeployContracts deploys Bridge, Relayer, ERC20Handler, ERC721Handler and CentrifugeAssetHandler and returns the addresses
 func DeployContracts(deployPK string, chainID uint8, url string, initialRelayerThreshold *big.Int) (*DeployedContracts, error) {
 
-	client, opts, err := accountSetUp(url, deployPK)
+	client, err := accountSetUp(url, deployPK)
```

Comment:
This deploy code has gone through a few iterations, I think at this point we should change the signature above to `DeployContracts(client *utils.Client, chainID....)`. This follows the pattern used elsewhere, and I think makes sense given the name.